br/pkg/lightning/backend/external/kv_reader.go

```diff
@@ -82,5 +82,6 @@ func noEOF(err error) error {
 }
 
 func (r *kvReader) Close() error {
+	r.byteReader.concurrentReader.largeBufferPool.Destroy()
```

Comment:
`largeBufferPool` is `nil` if the reader does not enable concurrent mode.

Reply:
https://github.com/pingcap/tidb/pull/47239/commits/90f68418d25be240ed2db79aad4bd33f488c5205